index.html

```diff
@@ -0,0 +1,71 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <link rel="stylesheet" href="style.css" type="text/css">
+    <meta name="viewport"
+          content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
+    <meta http-equiv="X-UA-Compatible" content="ie=edge">
+    <title>Document</title>
+</head>
+<body>
+    <div class="container">
+        <div class="photo"></div>
+        <div class="item">
+            <h3 class="item-name">1.Name</h3>
+            <p class="item-content">Yury Misiyevich</p>
+        </div>
+        <div class="item">
+            <h3 class="item-name">2.Contacts</h3>
+            <p class="item-content">
+                <a href="tel:+375 29 167 02 62">+375 29 167 02 62</a><br>
+                <a href="mailto:misiyevich@gmail.com">misiyevich@gmail.com</a>
+            </p>
+        </div>
+        <div class="item">
+            <h3 class="item-name">3.Summary</h3>
+            <p class="item-content">I want to change my life to better and i do my best for it!
+                I graduated Top 10% of my class at Belarusian National Technical University
+                at faculty of Power plant construction and engineering services
+                at  Heating, air conditioning, gas supply and air protection specialization
+                in 2016, and after working as engineer, I understood that I wanted something
+                more challenging. I started teaching myself coding 1,6 year ago paying an
+                average 3-4 hours a day! I am fast learner, good team player, ambitious and
+                self-motivated junior with good experience.</p>
+        </div>
+        <div class="item">
+            <h3 class="item-name">4.Skills</h3>
+            <p class="item-content">JavaScript, React+Redux, jQuery, Git, Python+Flask, SQL+phpLiteAdmin, HTML, CSS, C native</p>
+        </div>
+        <div class="item">
+            <h3 class="item-name">5.Code examples</h3>
+            <p class="item-content">
+                <a href="https://github.com/diesel167/taskManager">TaskManager</a><br>
+                <a href="https://github.com/diesel167/calendar">Calendar</a>
```

Comment:
It seems to me It would be more interesting if you deployed a project and showed a link to the final result.